components/Footer/Footer.js

```diff
@@ -69,7 +69,7 @@ export function Footer({}) {
                     <div>
                         <div>
                             <Link href="/">
-                                <img src={TestausserveriLogo.src} />
```

Comment:
This is a weird one. When using `.src`, it works in the Next.js app but doesn't in the Storybook. But if we remove it as you did, it'll work in the Storybook but stop working in the Next.js app.

<img width="1142" alt="image" src="https://user-images.githubusercontent.com/46541386/167267448-b2354b7f-9f78-4e70-8ced-cf32cf3d63cf.png">